application/templates/application/application-overview.html

```diff
@@ -25,36 +25,38 @@ <h5 style="margin: 1em; text-align: center;">
                     </table>
                 {% else %}
                     <h2>High School Application Form</h2>
-                    <form action="{% url 'dashboard:application:draftExistingApplication' 'ut_student' selected_app.pk %}"
-                        method="post" class="form-signup">
+                    <form action="{% url 'dashboard:application:draftExistingApplication' 'ut_student' selected_app.pk %}" prog-data-url="{% url 'dashboard:application:ajax_load_programs' %}" method="post" class="form-signup" style="padding-top: 10px;">
```

Comment:
I think since we've implemented sessions, we can get rid of these extra parameters from URLs referencing to the user_type.

Reply:
It will be resolved in the next PR which I will be creating by the end of the day